Dockerfile

```diff
@@ -1,3 +1,17 @@
+# Copyright 2019 The Kubernetes Authors.
```

Comment:
It seems weird to have boilerplate on the dockerfile

Reply:
There are boilerplate templates for `Dockerfile` and the python script, copied from capa (which was copied from another kubernetes repo), enforces this.  So 🤷‍♂